cfpb/consumerfinance.gov
# Header: Optionally include tagline in global-eyebrow template

Just noticed that the mega-menu Cypress test has an erroneous selector `a-tagline` instead of `.a-tagline`, and then realized that the tagline is never displayed in the mega menu, so why not just exclude it from the template entirely and make our header HTML smaller?

## Changes

- Make the tagline optional in the global-eyebrow template and exclude it from the mega menu.


## How to test this PR

1. Pull branch and see that there are only two `a-tagline`s in the markup (header and footer). Compare to prod, where there are three, because one is inside the mega menu, yet is never displayed.


## Screenshots

Before, three taglines in the markup:

<img width="602" alt="Screen Shot 2023-02-21 at 2 57 48 PM" src="https://user-images.githubusercontent.com/704760/220445838-5c41dd39-7531-4b70-b0c1-2a48b96aaf3c.png">

After, two taglines in the markup:

<img width="594" alt="Screen Shot 2023-02-21 at 2 58 02 PM" src="https://user-images.githubusercontent.com/704760/220445897-70f0ca55-04bb-41ea-8dd3-f71928e17b07.png">
 

## Notes and todos

- The global eyebrow and mega menu "eyebrow" are perhaps different enough that they should be made different templates entirely.
